### DISCUSSION TITLE
transaction not received

### DISCUSSION THREAD

**Discussion by Tony333KP** [Chia-Network/chia-blockchain#10599] [Support]
transaction at 02/12/2022 6:05:13 PM confirmed at height 1556708 but not received

**Comment by esaung** [comment]
can you provide more info (addresses) and debug logs?

**Comment by Tony333KP** [comment]
to : xch1ql9rt8hlu6c6c9rfk2ma4z2s43gugcqysqgcwtzaknwan5wl5t3s4ey7n4
